bridge/functions.php

```diff
@@ -112,9 +118,13 @@ function dd_wrap_autoloader()
     $composerAutogeneratedClass = null;
 
     dd_trace('spl_autoload_register', function () use (&$composerAutogeneratedClass) {
-
         $originalAutoloaderRegistered = dd_trace_forward_call();
-        list($loader) = func_get_args();
+        $args = func_get_args();
+        if (sizeof($args) == 0) {
```

Comment:
Nice change. Just a nit-pick - could we use the non-alias version of this function and do strict comparison? :)

```suggestion
        if (count($args) === 0) {
```